Makefile

```diff
@@ -156,6 +156,9 @@ clean:
 core-clean:
 	dune clean
 	rm -f bin
+	# we still need to keep the nonempty opam files in git for
```

Comment:
We could do this with `git clean -fx *.opam`.

Reply:
will be fixed in upcoming PR